llvm/test/CodeGen/RISCV/zilsd.ll

```diff
@@ -7,10 +7,9 @@
 define i64 @load(ptr %a) nounwind {
 ; CHECK-LABEL: load:
 ; CHECK:       # %bb.0:
-; CHECK-NEXT:    ld a2, 80(a0)
-; CHECK-NEXT:    ld zero, 0(a0)
-; CHECK-NEXT:    mv a0, a2
-; CHECK-NEXT:    mv a1, a3
```

Comment:
I believe this is the only saving showin in this PR - swapping a GPRPair copy for a GPR copy.

I will investigate further if there are savings on our benchmarks/codebases.